docs/concepts/event-subsriber.md

```diff
@@ -10,4 +10,21 @@ type: markdown
 
 Event Subscriber is an object that is subscribed to receive events.
 
-You can consider [Projection](/concepts/projection.md) as a type of Event Subscriber. 
\ No newline at end of file
+Here is a code example which shows how a method which handles this kind of messages looks like.
+
+  ```
+  final class TaskProjection
+      extends Projection&lt;TaskId, TaskItem, TaskItemVBuilder&gt; {
+      ...
+      @Subscribe
```

Comment:
In the text we speak about _one_ method. The source code sample has two. Please leave only the one which handles the `TaskCompleted`.

Reply:
LGTM